spec/controllers/challenge_claims_controller_spec.rb

```diff
@@ -20,40 +23,39 @@
 
     it 'will not allow you to see someone elses claims' do
       second_user = create(:user)
-      fake_login_known_user(user)
-      get :index, user_id: second_user.login
+      get :index, id: claim.id, user_id: second_user.id
       it_redirects_to_with_error(root_path, \
                                  "You aren't allowed to see that user's claims.")
     end
   end
 
   describe 'show' do
-    it 'redirects logged in user to the prompt' do
-      request_prompt = create(:prompt, collection_id: collection.id, challenge_signup_id: signup.id)
-      claim_with_prompt = create(:challenge_claim, collection: collection, request_prompt_id: request_prompt.id)
-      fake_login_known_user(user)
-      get :show, id: claim_with_prompt.id, collection_id: collection.name
-      it_redirects_to(collection_prompt_path(collection, claim_with_prompt.request_prompt))
+    it 'redirects' do
+      get :show, id: claim.id, collection_id: collection.name
+      it_redirects_to(root_path)
+    end
+
+    it 'needs a claim' do
+      get :show, collection_id: "none existent collection"
+      it_redirects_to_with_error(root_path, \
+                                 "What claim did you want to work on?")
     end
 
-    xit 'needs a collection' do
-      fake_login_known_user(user)
-      get :show
+    it 'needs a collection' do
```

Comment:
Descriptions need to describe how the code behaves in given contexts. You test that the code does `x` when it has a valid collection parameter and `y` when it doesn't have a valid collection parameter, not that the code needs a collection.